frontend/packages/dev-console/integration-tests/features/gitops/add-Manifest-URL-ODC-3067.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Fetch applications using Manifest URL and create secret
+    User should be able to fetch the applications using Manifest URL and create github secret
```

Comment:
It should be something like: User should be able to see list of applications managed by GitOps.

Reply:
Updated